fe/fe-core/src/main/java/org/apache/doris/nereids/NereidsPlanner.java

```diff
@@ -59,6 +69,18 @@
 public class NereidsPlanner extends Planner {
     public static final Logger LOG = LogManager.getLogger(NereidsPlanner.class);
 
+    static {
+        EventChannel.getDefaultChannel()
+                .addEnhancers(ImmutableList.of(new AddCounterEventEnhancer()))
+                .addConsumers(ImmutableList.of(
+                        new LogConsumer(CounterEvent.class, LOG),
+                        new LogConsumer(TransformEvent.class, LOG),
+                        new LogConsumer(EnforcerEvent.class, LOG),
+                        new LogConsumer(GroupMergeEvent.class, LOG),
+                        new LogConsumer(CostStateUpdateEvent.class, LOG),
+                        new LogConsumer(StatsStateEvent.class, LOG)));
```

Comment:
init channel in this class is weired. we need to init it in a event related class